glide.yaml

```diff
@@ -8,6 +8,8 @@ import:
   - instrument
 - package: github.com/uber-go/atomic
   version: ^1.2.0
+- package: github.com/google/codesearch
```

Comment:
could you use the m3db forked one, `github.com/m3db/codesearch`

Reply:
sure, wasn't aware we had a fork